SIT/test.sh

```diff
@@ -164,18 +169,19 @@ for testDir in $(ls -d ${PHASE}/*); do
   if [ $rtn -eq 1 ]; then
     _error "${testDir} files differ (expected vs actual):"
     sdiff -w 200 ${testDir}/expected.out ${testDir}/output/actual.out
+    _captureOutput
     errors=1
     continue
   elif [ $rtn -ne 0 ]; then
     _error "${testDir} had some other problem running diff"
+    _captureOutput
     errors=1
     continue
   fi
 
   _info "PASS: ${testDir} returned expected results"
 done
 if [ $errors -ne 0 ]; then
-  _captureOutput
```

Comment:
Having it here was only capturing the last run's output alone. Now, it will capture whenever there is a failure.